middleware/keyauth/config.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/gofiber/fiber/v3"
 )
 
+type KeyauthKeyLookupFunc func(c fiber.Ctx) (string, error)
```

Comment:
Since its package is keylookup, there's no need to repeat it in the typedef.
```suggestion
type KeyLookupFunc func(c fiber.Ctx) (string, error)
```